runtime/lua/vim/lsp/util.lua

```diff
@@ -1875,7 +1875,7 @@ function M.make_position_params(window, offset_encoding)
   local buf = api.nvim_win_get_buf(window)
   if offset_encoding == nil then
     vim.notify_once(
-      'warning: offset_encoding is required, using the offset_encoding from the first client',
+      'warning: offset_encoding is required in vim.lsp.util.make_position_params, using the offset_encoding from the first client',
```

Comment:
```suggestion
      'warning: vim.lsp.util.make_position_params position_encoding param is required; using the offset_encoding from the first client',
```

Will need to rebase after https://github.com/neovim/neovim/pull/31286